model/cached_feed.py

```diff
@@ -0,0 +1,201 @@
+# encoding: utf-8
+from nose.tools import set_trace
+import datetime
+import logging
+import time
+from sqlalchemy.sql.expression import (
+    and_,
+)
+from sqlalchemy import (
+    Column,
+    Date,
+    DateTime,
+    ForeignKey,
+    Index,
+    Integer,
+    Unicode,
+    UniqueConstraint,
```

Comment:
A lot of the time these basic imports aren't necessary -- they're used in the big model.py but not in these smaller files. Since this will happen many tiemes, the simplest way to fix this sort of problem will be to run this code through pylint afterwards.